### DISCUSSION TITLE
Is it possible to improve threat detection with other or improved feeds?

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13460] [Unsupported Versions]
Hello,

I'm a former QRadar Analyst. I'm considering recommending Security Onion as a cost-effective SIEM solution for small businesses that can't afford commercial products like Splunk or QRadar.

I'm familiar with QRadar's tiered pricing model, which offered varying levels of rule updates. I'm wondering if Security Onion, as a free open-source solution, provides sufficient rule coverage for effective threat detection.

I'm also interested in exploring options for purchasing enhanced rule sets or subscription services to supplement Security Onion's capabilities. Are there any recommended commercial or open-source options for get more accurate and up-to-date rules for Suricata or Zeek? Does it make sense?

Any insights or experiences would be greatly appreciated.

Thank you for your insights.

**Comment by dougburks** [comment]
> I'm familiar with QRadar's tiered pricing model, which offered varying levels of rule updates. I'm wondering if Security Onion, as a free open-source solution, provides sufficient rule coverage for effective threat detection.

Yes, our free platform includes the free Emerging Threats Open (ETOPEN) ruleset and many organizations find that combination quite effective.

You can read more about ETOPEN at:
https://docs.securityonion.net/en/2.4/nids.html#etopen

You can see some examples of its effectiveness in our Quick Malware Analysis blog posts. Here is the most recent:
https://blog.securityonion.net/2024/07/quick-malware-analysis-latrodectus.html

> I'm also interested in exploring options for purchasing enhanced rule sets or subscription services to supplement Security Onion's capabilities. Are there any recommended commercial or open-source options for get more accurate and up-to-date rules for Suricata or Zeek? Does it make sense?

You can choose to upgrade to the paid Emerging Threats Pro (ETPRO):
https://docs.securityonion.net/en/2.4/nids.html#etpro

There are other options listed on that page, but ETOPEN and ETPRO are the most common on our platform.

In addition to network detection (NIDS), Security Onion also includes log detection via Sigma and file detection via YARA:
https://docs.securityonion.net/en/2.4/sigma.html
https://docs.securityonion.net/en/2.4/yara.html
